src/CallMessage.ts

```diff
@@ -13,34 +14,31 @@ import type { NativeCallMessageInfo } from './type/CallMessage';
  * Provides access to information about a callMessage, including the call
  * message content, contentType, messageType, and voiceEventSid
  *
- * @remarks
- * Note that the callMessage information is fetched as soon as possible from the
- * native layer, but there is no guarantee that all information is immediately
- * available. Methods such as `CallMessage.getContent` or `CallMessage.getSid`
- * may return `undefined`.
- *
  * @public
  */
-export class CallMessage {
+export class CallMessage extends EventEmitter {
   /**
-   * Content body of the message
+   * The content of the message which should match the contentType parameter.
    */
-  public _content: string;
+  private _content: any;
 
   /**
-   * MIME type for the message
+   * The MIME type of the content. application/json and is the only contentType
```

Comment:
I think we should leave the description as `The MIME type of the content.` and leave the rest out. We can document what is supported in the `CallMessage.ContentType` enum.